README.md

```diff
@@ -11,6 +11,8 @@ Fetch or emit events to Datadog.
 
 * `filter`: *Optional.* Regexp to filter events by title when checking.
 
+* `datadog_host`: *Optional.* Datadog host url to switch from US to Europe datacenter, default to US.
```

Comment:
@vito @pivotal-izabela-gomes I wonder if it would be preferable this to be a flag to enable EU datacenter instead of passing the host. The host should not change and it would make a better experience for the user?

Reply:
Hey @dcsg, I was looking at the go-datadog-api and they also default to the US api and just recommend to set the env variable to override it with the eu datacenter. I'd personally like to mimic the same behavior just for consistency and also, I don't think it would impact the user experience that much.